File: El Mamoun/Lv 1/Problem 20.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
Problem 20: Write a program to calculate Circle area inscribed in a square, then print it on the screen.
 The use should enter:
 A
*/
namespace Problem_20
{
    internal class Program
    {

        static float ReadAreaSquare()
        {

            Console.WriteLine("Pleae Enter a Area Square?: ");
            float AreaSquare = Convert.ToSingle(Console.ReadLine());

            return AreaSquare;
        }
        static double CalRectangeAreaInscribedSquare(double Square)
        {

            const double PI = 3.14159265359;
            double Area = (PI * (Math.Pow(Square, 2)) / 4);

            return Area;
        }
        static void PrintArea(double Area)
        {
            Console.WriteLine("The Area of Circle is: " + Area);
        }

        static void Main(string[] args)
        {

            PrintArea(CalRectangeAreaInscribedSquare(ReadAreaSquare()));
            Console.ReadKey();
        }



    }
}

File: El Mamoun/Lv 1/Problem 20.cpp
#include<iostream>
using namespace std;

/*
Problem 20: Write a program to calculate Circle area inscribed in a square, then print it on the screen.
 The use should enter:
 A
*/

float ReadSquareSide()
{
	float A;

	cout << "\nPlease Enter A? ";
	cin >> A;

	return A;
}

float CircleAreaThroughInscribedSquare(float A)
{
	const float PI = 3.1415929;

	float Area = (PI * pow(A, 2)) / 4;

	return Area;
}

void PrintResult(float Area)
{
	cout << "\n Area Of Circle is = " << Area << endl;
}


int main()
{

	PrintResult(CircleAreaThroughInscribedSquare(ReadSquareSide()));

	return 0;
}